scripts/dataset_processing/tts/audio_processing/preprocess_audio.py

```diff
@@ -173,9 +169,9 @@ def main(cfg):
         output_durations += output_duration
         output_entries.append(output_entry)
 
-    write_manifest(manifest_path=output_manifest_path, entries=output_entries)
+    write_manifest(output_path=output_manifest_path, target_manifest=output_entries, ensure_ascii=False)
     if filter_file:
-        write_manifest(manifest_path=filter_file, entries=filtered_entries)
+        write_manifest(output_path=str(filter_file), target_manifest=filtered_entries, ensure_ascii=False)
```

Comment:
Possible to make write manifest take in str and Path and just call str(input) inside it to avoid this style

Reply:
Good point! will do.